lib/gems/pending/util/miq-password.rb

```diff
@@ -72,6 +73,40 @@ def self.encrypted?(str)
     !!split(str).first
   end
 
+  def self.encrypt_hash(values)
+    hash_values = values.deep_clone
+
+    hash_values.select { |f, _| f =~ /[Pp]assword/ }.each do |key, val|
+      next if val.strip.empty?
+
+      key_in_symbol = key.kind_of?(Symbol)
+      hash_values.delete(key)
+
+      key = "password::#{key}" unless key =~ /password::/
```

Comment:
This is a slippery slope, knowing what is a password.

Wonder if it makes sense to look into `Vmdb::Settings::Walker` - possibly moving that walker into here